Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -112,8 +119,28 @@ internal struct ReflectionServiceData: Sendable {
               """
           )
         }
+        if self.fieldNumbersByType[typeName] == nil {
+          self.fieldNumbersByType[typeName] = []
+        }
+        let numberPosition = self.fieldNumbersByType[typeName]!.count
+        self.fieldNumbersByType[typeName]?.insert(
+          `extension`.number,
+          at: numberPosition
+        )
       }
+      // Populating messageTypeNames array.
+      self.messageTypeNames.append(
+        contentsOf: fileDescriptorProto.qualifiedMessageTypes
+      )
+    }
+  }
+
+  internal static func extractTypeNameFrom(fullyQualifiedName name: String) -> String {
+    var nameCopy = name
+    if nameCopy.first == "." {
+      nameCopy.removeFirst()
     }
+    return nameCopy
```

Comment:
We don't need a whole function for this, it's just a one liner:`String(name.drop(while: { $0 == "." }))`